deploy/helm/flightctl/charts/keycloak/values.yaml

```
@@ -1,37 +1,64 @@
-keycloak:
-  rootUrl: https://flightctl.example.com:443
-  redirectUris:
-    - https://flightctl.example.com:443/*
-  authority: http://keycloak:8080/realms/flightctl
-  clientid: flightctl-ui
-  redirect: https://ui.flightctl.example.com:443
-  secret: CHANGE_ME
-  namespace: flightctl-external
-  image: quay.io/keycloak/keycloak:latest
-  keycloakAdmin: admin
-  keycloakAdminPassword: 12345678-1234-1234-1234-1234
-  db:
-    namespace: flightctl-internal
-    image: quay.io/cloudservices/postgresql-rds:12-9ee2984
-    pvc:
-      labels: {
-        app: keycloak,
-      }
-      annotations: {
-        kubernetes.io/reclaimPolicy: Delete
-      }
-      accessModes: 
-        - ReadWriteOnce
-      size: 1Gi
-      storageClassName: standard
-      volumeMode: Filesystem
-    PGPASSWORD: CHANGE_ME
-    postgresqlDatabase: keycloak
-    postgresqlMasterPassword: CHANGE_ME
-    postgresqlMasterUser: admin
-    postgresqlPassword: CHANGE_ME
-    postgresqlUser: keycloak
-    dbAddr: jdbc:postgresql://keycloak-db/keycloak
-    dbVendor: postgres
+# Copyright Red Hat, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
 
+## @param namespace The namespace into which to deploy Keycloak
+namespace: flightctl
```

```suggestion
namespace: flightctl-external
```

let's make sure we provide the separation of keeping externally exposed services in another namespace

---

I was thinking we should change the default to a single namespace for a better first-user experience and to follow the Helm best practice to let the user decide the namespace and whether Helm shall create it in the `helm upgrade --install` command. For the PaaS, we can then use a `values.yaml` override?